ExampleMod/Content/Tiles/Furniture/ExampleDoorOpen.cs

```diff
@@ -35,9 +37,8 @@ public class ExampleDoorOpen : ModTile
 			AddMapEntry(new Color(200, 200, 200), Language.GetText("MapObject.Door"));
 
 			// Placement
-			// In addition to copying from the TileObjectData.Something templates, modders can copy from specific tile types. CopyFrom won't copy subtile data, so style specific properties won't be copied, such as how Obsidian doors are immune to lava.
-			TileObjectData.newTile.CopyFrom(TileObjectData.GetTileData(TileID.OpenDoor, 0));
-			/* This is what is copied from the OpenDoor tile
+			// The TileID.OpenDoor TileObjectData is incorrect, as they sometimes are, so we will not be copying from it in this case
```

Comment:
Incorrect how? This implies a bug in vanilla. If there's no vanilla bug, it may be better to explain why it doesn't apply to our tile/spritesheet.

Reply:
I updated this to be a bit clearer.

These aren't "technically" vanilla bugs because vanilla doesn't use anchors after placement and you can't place open doors. The layout of DoorOpen tile is also incorrect because the alternates are supposed to be in-line with styles. This was likely changed in an effort to fix doors into the 2048 pixel limit in vanilla, but the fix does not fit with the default TileObjectData behaviors we use in tModLoader. This too, however, doesn't count as a bug because the vanilla framing checks are all hardcoded as well.

I don't think it helps to explain too deeply why this and ExampleSign aren't using the vanilla TOD, as the whole situation can be confusing to a modder, but I think the new comments help explain it.